packages/fxa-auth-server/config/index.ts

```diff
@@ -2106,4 +2106,4 @@ if (convictConf.get('isProduction')) {
 export type conf = typeof convictConf;
 export type ConfigType = ReturnType<conf['getProperties']>;
 
-module.exports = convictConf;
+export default convictConf;
```

Comment:
nit: Since we have to add .default to all the requires anyways,  it might be cleaner to rename convictConf to config, and then just 
`export config`. This would switch imports to: `const { config } = require('../config')`. Never been a fan of having to sue the `.default`. maybe that's just me though.

Reply:
For some the `require`s that certainly would work and I can update them.  But more often than not `config` is assigned the return value of `getProperties` of the exported object.  I don't think there's a way to destruct and chain the function call?